### ISSUE TITLE
Similar executables found (uvision project files)

### ISSUE THREAD

**Issue by 0xc0170** [ARMmbed/valinor#3]
When yt debug is executed the first time, it creates executables (project files). This is related to uvision files, the second time it prints an error and fails to open:

```
 yt debug build\frdm-k64f-gcc\test\some_test
error: build\frdm-k64f-gcc\test\some_test is similar to several executables found. Please use an exact name:
some_test.uvgui.my_name
some_test.uvopt
some_test.uvproj
```

**Comment by 0xc0170** [comment]
I am adding here, there's a problem for mkit for example, where hex are generated. Valinor reports that I should be more specific, using as executable: minar-test-complex_dispatch, and there's minar-test-complex_dispatch-combined.hex

**Comment by autopulated** [comment]
I think the problem here is actually that valinor is renaming the elf-file (with no extension) to elf-file.axf, so that MDK can load it. That means yotta debug doesn't find an exact match for the file name when you run it a second time. So copying instead of renaming the elf file should help.

**Comment by autopulated** [comment]
I think this should have been fixed by #4

**Comment by rainierwolfcastle** [comment]
http://jira-dev0.cambridge.arm.com/browse/IOTSFW-857

**Comment by rainierwolfcastle** [comment]
ARM Internal Ref: IOTSFW-996

**Comment by autopulated** [comment]
Closing, assumed fixed by #4. Please re-open if this is still a problem.
